.gitmodules

```diff
@@ -3,7 +3,7 @@
 	url = git://github.com/galaxyproject/ansible-galaxy-tools.git
 [submodule "roles/galaxyprojectdotorg.galaxy-extras"]
 	path = roles/galaxyprojectdotorg.galaxy-extras
-	url = git://github.com/galaxyproject/ansible-galaxy-extras
+	url = git://github.com/mvdbeek/ansible-galaxy-extras
```

Comment:
These are defining the submodule urls. When you click on artimed's role/galaxyproject.org-ansible-galaxy-extras folder (which is a submodule) you may get to my fork or to the galaxyproject fork, depending on this. This also determines which remotes are available for your local repo. The full doc is [here](https://git-scm.com/book/en/v2/Git-Tools-Submodules)